src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -190,6 +193,12 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
       });
   }
 
+  formatConceptDOI() {
```

Comment:
I don't feel strongly about this one, but since we will be going in and changing much of the guts of DOIs in this release, this does seem like a good time to fix it.

Reply:
Closing this ticket as I moved the fix to the webservice https://github.com/dockstore/dockstore/pull/5894